### DISCUSSION TITLE
Changing auto1 & auto0 by Mqtt?

### DISCUSSION THREAD

**Discussion by RaW79** [revk/xESP32-Faikout#553] [Q&A] [answered]
Hi

I´m struggeling a lot setting auto0 & auto1 by Mqtt. 
autot is working fine with 
topic: command/GuestAC
payload: {"autot":22}
or
topic: setting/GuestAC
payload: {"autot":22}

But auto1&auto0 not. also tried with variants like
topic: command/GuestAC/control
payload: {"auto0":22:00}

But no success. Kindly ask for your help.
Thx

**Comment by revk** [comment, marked as answer]
The help should say *HHMM* (not *HH:MM*), and `22:00` is not valid JSON, use `2200`. Indeed, even `0700` is not valid JSON so use `700` or `"0700"` where there is a leading zero.

**Reply by RaW79** [reply]
Thanks. Can you please inform how it should be send?

command/...
or
setting/....

...../GuestAC
or
...../GuestAC/control

**Reply by revk** [reply]
`setting/NameOfDevice` with payload `{"auto0":2200}`

**Reply by RaW79** [reply]
Big Thx. Finally afters hours of testing (mainly struggled because of the data format in the payload) it works :)

**Reply by revk** [reply]
You can also set using the web interface, obviously.

**Reply by RaW79** [reply]
Sure. But intention is to make it operatable for everyone inside our house. Grandparents can not use a webinterface and therefore created this touch display: https://github.com/revk/ESP32-Faikin/discussions/490 

See photo 3 which is for operation of auto0 & auto1
